game/src/edu/pucrs/br/player/Players.java

```diff
@@ -71,4 +72,35 @@ public void listItensByPrice() {
             System.out.println();
         }
     }
-}
\ No newline at end of file
+    public void SearchItem(String nome){
+        String term = nome.toLowerCase();
+        boolean found = false;
+        if (players.isEmpty()) {
```

Comment:
Necessário adicionar um return apos esse sout para evitar que prossiga para o for abaixo quando não houver jogadores;